src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -805,6 +811,23 @@ private static bool IsFieldLikeEventAccessor(MethodSymbol method)
                 ((EventSymbol)associatedPropertyOrEvent).HasAssociatedField;
         }
 
+        private void CompileDeletedMethods(NamedTypeSymbol containingType, TypeCompilationState compilationState)
+        {
+            Debug.Assert(_moduleBeingBuiltOpt != null);
+            Debug.Assert(_moduleBeingBuiltOpt.EncSymbolChanges != null);
+
+            var methodsToDelete = _moduleBeingBuiltOpt.EncSymbolChanges.GetDeletedMethods(containingType.GetCciAdapter());
+            foreach (var deletedMethod in methodsToDelete)
+            {
+                var methodSymbol = deletedMethod.GetISymbol().GetSymbol<MethodSymbol>();
+                var method = new SynthesizedDeletedMethod(methodSymbol, containingType);
+
+                method.GenerateMethodBody(compilationState, _diagnostics);
+
+                _moduleBeingBuiltOpt.AddSynthesizedDefinition(containingType, method.GetCciAdapter());
```

Comment:
Is there a better way to do this than calling both methods? Seemed like other things don't, but this is what I needed to be able to see the emitted method in the `DeltaMetadataWriter` (and for it to have a body!)